plugins/inputs/snmp_trap/snmp_trap_test.go

```diff
@@ -1293,19 +1288,13 @@ func TestReceiveTrap(t *testing.T) {
 				PrivProtocol: tt.privProto,
 				PrivPassword: tt.privPass,
 			}
-			require.Nil(t, s.Init())
-			// Don't look up oid with snmptranslate.
-			s.execCmd = fakeExecCmd
+
+			require.NoError(t, s.Init())
+
```

Comment:
Oops, it looks like s.Init() was there originally. We must have removed it before we found it was definitely necessary and put it back :)